packages/test/test-end-to-end-tests/src/test/containerDirtyFlag.spec.ts

```diff
@@ -63,7 +66,10 @@ describeCompat("Container dirty flag", "NoCompat", (getTestObjectProvider, apis)
 
 		await args.ensureSynchronized();
 		await args.opProcessingController.pauseProcessing(container);
-		assert(toDeltaManagerInternal(dataStore.runtime.deltaManager).outbound.paused);
+		const deltaManagerInternal = toDeltaManagerInternal(dataStore.runtime.deltaManager);
+		if (isIDeltaManagerInternal(deltaManagerInternal)) {
```

Comment:
See my prior comment on naming. Usage of "internal" on these two lines does not make sense - we already sort of got "internal" object, why do we need to check that it's "internal"? In reality, first line does not return internal interface, even though it feels that way from the name of API

Reply:
Probably could be resolved by updating `toDeltaManagerInternal` to return `IDeltaManagerInternal`.

Reply:
Yeah, I didn't realized we already have a method name IDeltaManagerInternal which actually returns IDeltaManager and not IDeltaManagerInternal.
So, I don't think we can make toIDeltaManagerInternal return IDeltaManagerInternal as it actually wants to expose IDeltaManager. So, I think I should rename IDeltaManagerInternal to IDeltaManagerFull which sorts of represents that it is full interface exposing some other props as well.

Reply:
toDeltaManagerInternal is legacy and alpha. Also, I renamed IDeltaManagerInternal to IDeltaManagerFull
 which represents that it has all/extra props. IDeltaManagerInternal is expose publicly by shared Object.

Reply:
👍I think I must have looked at toDeltaManagerErased tagging.